proto/lookout/sdk/service_data.proto

```diff
@@ -41,40 +42,69 @@ message File {
     string hash = 3;
     // Raw content of the file.
     bytes content = 4;
-    // UAST.
+    // UAST is a Babelfish v1 UAST.
     gopkg.in.bblfsh.sdk.v1.uast.Node uast = 5 [(gogoproto.customname) = "UAST"];
     // Programming/data/markup language of the file.
     string language = 6;
 }
 
+// Change contains two version of a File in a revision range.
 message Change {
     option (gogoproto.sizer) = false;
     option (gogoproto.protosizer) = true;
     
+    // Base is the file version at the base of the revision range.
     File base = 1;
+    // Head is the file version at the head of the revision range.
     File head = 2;
 }
 
+// ChangesRequest defines a request of Changes to the Data service.
 message ChangesRequest {
+    // Base of the revision range.
     ReferencePointer base = 1;
+    // Head of the revision range.
     ReferencePointer head = 2;
+    // IncludePattern is a regexp. Only the files with matching filenames will
+    // be included.
     string include_pattern = 3;
+    // ExcludePattern is a regexp. Any files with matching filenames will be
+    // excluded.
     string exclude_pattern = 4;
+    // ExcludeVendored will exclude any files inside a vendor path.
     bool exclude_vendored = 5;
+    // WantContents will fill the response file Content.
     bool want_contents = 6;
+    // WantUAST will fill the response file UAST and Language.
     bool want_uast = 7 [(gogoproto.customname) = "WantUAST"];
+    // WantLanguage will fill the response file Language.
     bool want_language = 8;
+    // IncludeLanguages will filter files by language. The language names are
+    // case insensitive. The list of language names is available at
+    // https://github.com/github/linguist/blob/master/lib/linguist/languages.yml
```

Comment:
not sure if it's the best possible link. Enry is 8 months behind linguist master and doesn't include some languages from that list. Though they aren't very important.

Reply:
But is there a clean file like this we can link from the enry repo?

Reply:
there are different files. The cleanest one, in my opinion, is https://github.com/src-d/enry/blob/master/data/type.go

I'm okay with the current link to linguist. It is just a note in case you weren't aware of it.